object-cache.php

```diff
@@ -278,6 +306,16 @@ function delete( $id, $group = 'default' ) {
 		return $result;
 	}
 
+	public function delete_multiple( array $keys, $group = '' ) {
+		$values = array();
+
+		foreach ( $keys as $key ) {
+			$values[ $key ] = $this->delete( $key, $group );
```

Comment:
https://www.php.net/manual/en/memcached.deletemulti.php

Reply:
Same - there are no `multi` methods in Memcache.

Reply:
There is an [undocumented feature](https://github.com/websupport-sk/pecl-memcache/blob/NON_BLOCKING_IO_php8/src/memcache.c#L1053) that `Memcache::delete()` accepts an array as its `$key` parameter; however, its implementation is incompatible with WordPress's expectations because `delete()` [does not return an array](https://github.com/websupport-sk/pecl-memcache/blob/NON_BLOCKING_IO_php8/src/memcache.c#L1056-L1059) and, therefore, there is no way to tell which values have been deleted.

Reply:
Could we not do a multi delete and a multi get? If you are delete 20 keys, is a better to one multiple delete and one multiple get than 20 calls.